src/lib/clamp.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Given a number, a minimum value, and a maximum value,
+ * Return the minimum value if the number is less than the minimum,
+ * or the maximum value if the number is greater than the maximum,
+ * or the number itself if it is greater than the minimum or less than the maximum.
+ */
+export default function clamp(
```

Comment:
Added this function to ensure `heading` blocks are rendered with a valid style. See `views/product-landing/components/heading/index.tsx` for more info.

Reply:
✅  Great call - added a short test suite to cover the cases, realized we probably want to throw an error if the provided `min` value is greater than `max` 👍

Reply:
Thank you! And the error message is a great addition, nicely done ✨